resources/local/chainspec.toml.in

```diff
@@ -163,6 +163,10 @@ maximum_round_length = '17 seconds'
 # Expressed as a fraction (1/5 by default).
 reduced_reward_multiplier = [1, 5]
 
+[highway.performance_meter]
+# The number of recent blocks to consider when measuring performance for the purpose of deciding the round length.
+blocks_to_consider = 10
```

Comment:
This was related to failed attempts at improving Highway round exponent changing behavior - it's an old thing and shouldn't be reintroduced.

Reply:
Removed in 080a969198464c157308f2b8dffd72450c0aece4